presto-iceberg/src/test/java/org/apache/iceberg/rest/IcebergRestCatalogServlet.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.iceberg.rest;
+
+import com.facebook.airlift.log.Logger;
+import org.apache.hc.core5.http.ContentType;
+import org.apache.hc.core5.http.HttpHeaders;
+import org.apache.iceberg.exceptions.RESTException;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.relocated.com.google.common.io.CharStreams;
+import org.apache.iceberg.rest.RESTCatalogAdapter.HTTPMethod;
+import org.apache.iceberg.rest.RESTCatalogAdapter.Route;
+import org.apache.iceberg.rest.responses.ErrorResponse;
+import org.apache.iceberg.util.Pair;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.UncheckedIOException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+
+/**
+ * The RESTCatalogServlet provides a servlet implementation used in combination with a
+ * RESTCatalogAdaptor to proxy the REST Spec to any Catalog implementation.
+ */
+public class IcebergRestCatalogServlet
+        extends HttpServlet
+{
+    private static final Logger LOG = Logger.get(RESTCatalogServlet.class);
```

Comment:
We can config the logger in `IcebergQueryRunner.setupLogging()`, using `logging.setLevel("org.apache.iceberg", OFF)` to ignore the error message.

Reply:
I've gone ahead and updated the handling for the test logging that takes into account some of the NITs but also re-works what messages are logged. Take a look and let me know what you think! 

Essentially, I'm suppressing the stack trace in the case of the load table requests because this endpoint is called often to check on the existence of a table. In all other cases, the errors/stack traces are shown for debugging purposes. I'm hesitant to completely blanket-ignore all error messages, but definitely could do something like `logging.setLevel("org.apache.iceberg.rest", OFF)` if we think that would be best!

Reply:
Sure, I'm OK on the approach. Just want to get rid of the unnecessary error message.